libexec/probes/worker-scripts/uploader/rabbitmquploader.py

```diff
@@ -44,21 +44,28 @@ def publishSToMq(self, arguments, event_types, summaries, summaries_data):
     def SendMessagetoMQ(self, msg_body):
         # the max size limit in KB but python expects it in bytes                                                                           
         size_limit = self.maxMQmessageSize * 1000
-        size_msg = sys.getsizeof(json.dumps(msg_body))
+        size_msg = self.total_size(msg_body)

```

Comment:
This one is pretty bizarre.  Since `msg_body` is a string, why not do `len(msg_body)`?

Reply:
msg_body is a python dict so after doing some research and having lots of problems I found out this is the most reliable way to check for size.

Reply:
but we eventually convert it to a JSON object, right?  Seems like we're doing the work twice - might as well convert to JSON here and then reject; else, keep it below.